tools/expotools/src/commands/UpdateVendoredModule.ts

```diff
@@ -503,6 +503,29 @@ async function askForModuleAsync(): Promise<string> {
   return moduleName;
 }
 
+async function getPackageJsonPathsAsync(): Promise<string[]> {
+  const packageJsonPath = path.join(Directories.getAppsDir(), '**', 'package.json');
+  return await glob(packageJsonPath, { ignore: "**/node_modules/**" });
+}
+
+async function updateDependencyAsync(packgeJsonPath: string, dependencyName: string, newVersion: string): Promise<boolean> {
+  const jsonFile = new JsonFile(packgeJsonPath);
+  const packageJson = await jsonFile.readAsync();
+  
+  let dependencies = ((packageJson || {}).dependencies || {});
```

Comment:
Why `let`? And I think we can assume that `packageJson` is at least an empty object 😉 We can also say that if packageJson doesn't contain `dependencies` block we don't have to continue and go to the `if` block below.